cirq/experiments/random_quantum_circuit.py

```diff
@@ -0,0 +1,229 @@
+from typing import (Callable, Container, Dict, Iterable, List, Sequence,
+                    TYPE_CHECKING, Tuple)
+
+from cirq import circuits, devices, google, ops, value
+from cirq._doc import document
+
+if TYPE_CHECKING:
+    import numpy as np
+    import cirq
+
+# A set of 2-qubit interactions that can be performed simultaneously on a grid.
+TwoQubitInteractionLayer = Container[
+    Tuple[devices.GridQubit, devices.GridQubit]]
+
+
+class GmonLayer(TwoQubitInteractionLayer):
```

Comment:
I'd remove "Gmon" from the name since this could conceivably be used with other devices as long as they provide a full grid of qubits. Perhaps `GridLayer`?

Reply:
I renamed it to `GridInteractionLayer` and deleted `TwoQubitInteractionLayer`.